app/controllers/CalculationListController.scala

```diff
@@ -42,7 +46,11 @@ class CalculationListController @Inject()(val authentication: AuthenticationPred
         logger.error(s"Invalid tax year '$taxYearEnd' received in request.")
         Future.successful(BadRequest(Json.toJson[Error](InvalidTaxYear)))
       } else {
-        getCalculationList(nino, taxYearEnd)
+        if(appConfig.isHIPFeatureSwitchEnabled("get-legacy-calc-list-1404")) {
```

Comment:
will there me more apis we need to add feature switches for?    
If so can we start off using enums instead of strings for feature switching?